app/controllers/users_controller.rb

```diff
@@ -1,4 +1,30 @@
 class UsersController < ApplicationController
+
+  def show
+    @user = User.find params[:id]
```

Comment:
- using `find_by` instead of `find` method
- what is difference between `find_by` and `find`
- Check case not found user